### PR TITLE
week 2 Project-news-site

### PR DESCRIPTION
**Pull request by Mimmisen**
My week 2 project news site 
https://clever-villani-3aab16.netlify.app

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,213 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  margin: 0;
+  background: white
```
Comment (emmahogberg88) [inline code/style.css:3]: if you put your font-family in the body you don´t need to declare it for all headings and so on and you save yourself some rows of coding :)

File: code/style.css [hunk 2]
```diff
@@ -1,4 +1,213 @@
...
+@media (min-width: 668px) and (max-width: 1023px) {
+.news-box {
+ width: 300px; 
+ }
+}
+
+
+@media (min-width: 1024px) {
+.news-box {
+  width: 350px;
```
Comment (emmahogberg88) [inline code/style.css:182]: I think this is "tossed around" and may have caused you some headache?  If you changes the width property for your .nexs-box (row 95) to 350px and in the media query for bigger screens, set the width to 550px and then it works perfectly!

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,213 @@
...
+ font-size: 110px; 
+ margin: 0 0 50px 0;
+ }
+}
+
+
+@media (min-width: 668px) {
+.small-news {
+ flex-flow: wrap; 
+ }
```
Comment (emmahogberg88) [inline code/style.css:169]: If you want you can use one media query for the same screen sizes, then you save yourself some rows of code :) like:
@media (min-width: 668px) {
  header {
    justify-content: space-around;
   flex-direction: column;
}
  nav {
    align-self: center; 
  }

  and so on....
}

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,213 @@
...
+  background-color: white;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  object-fit: none;              
+  align-items: center;
+}
+
+
+.news-box {
```
Comment (emmahogberg88) [inline code/style.css:95]: I left another comment in the end (row 182). A suggestion is to set a max-height on .news-box {max-height: XXXpx; width: 350px; object-fit: cover;} and add another width for your media query on bigger screens, then the images will looks reallly good on all screens :)

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,213 @@
...
+
+.big-news {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+
+.big-news-img {
```
Comment (emmahogberg88) [inline code/style.css:67]: The picture gets very big on large screens. A suggestion is to add a view height (vh) for the .big-news-img so the entire image and heading is shown when you enter the site (max-height: 55vh; object-fit:cover;) 
(maybe only add this for a media query).

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,213 @@
...
+
+
+h4 { /*text for footer*/
+  font-family: 'Bebas Neue', cursive;
+  color: #FFF8F0;
+}
+
+
+@media (min-width: 668px) {
+  header {
```
Comment (emmahogberg88) [inline code/style.css:144]: The logo get stuck to the top on big screens. Perhaps you can add some padding-top?

File: code/style.css [hunk 7]
```diff
@@ -1,4 +1,213 @@
...
+
+h1 {   
+  color: #A68D65;
+  margin: 10px;
+  font-family: 'Bebas Neue', cursive;
+  font-size: 88px;
+}
+
+
+.small-news {
```
Comment (emmahogberg88) [inline code/style.css:85]: The images with text jumps around a bit. If you want you can use .small-news {align-items: flex-end;} the pictures stays in places regardless of their sizes and the text under the images are aligned :)

File: code/style.css [hunk 8]
```diff
@@ -1,4 +1,213 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
```
Comment (emmahogberg88) [inline code/style.css:1]: You have a good structure in your css file. It´s easy to find what you are looking for!

**Review by emmahogberg88 — COMMENTED** [review]
Good work Mimmi! I really enjoyed reviewing your code and all requirements are fulfilled :) 

Your site looks very nice! I specially like the hover effect on the navbar, the logo and the background of the image in the big-news section, it is very nice! 

Your navbar and page looks really good on mobile. 

I left some comments/suggestions/questions. I hope it can be helpful :)
Just ask if you have any questions about the comments!

/Emma

File: code/index.html [hunk 9]
```diff
@@ -2,21 +2,66 @@
...
+        <a href="#home">Home</a>
+        <a href="#news">News</a>
+        <a href="#contact">Contact</a>
+        <a href="#about">About</a>
+      </div>
+     </nav>
     </header>
+    
 
+    <article>
```
[the remark this answers is not in the document]
Comment (Mimmisen) [reply code/index.html:28]: I can't see what you are trying to show after use? I did the same when I reviewed and for some reason, github doesn't like it when you try to show an example. Can you write it in another way? :)

[the remark this answers is not in the document]
Comment (emmahogberg88) [reply code/index.html:28]: Oh, thats good to know :) I saw this on a site and it says his helps with search optimization for your site: "The <body> content is divided with three semantic tags <nav>, <main> and <footer>." So I think you can use main instead om article to wrap your main content if you like

**Pull request closed without merging**